### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by wwwdrt**
<h1 align="center">Hey CRs</h1>

<div align="center">
Please feel free to reach out to me on Slack anytime. You can send me a direct message or give me a call. If you choose to call, please introduce yourself so I know who I'm speaking with. I'm here to help, so don't hesitate to get in touch 😁
</div>

<br>

<div align="center">
  

### Latest project 🔥"[AI Quiz Generation Platform](https://github.com/wwwdrt/quizzen)"
</div>

<h1 align="center">Changes</h1>

<div align="center">
In this update I added tailwindcss and implemented the Calculator component as per design request.
</div>

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @wwwdrt, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the linters passed, kindly fix it

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+const Calculator = () => (
+  <div className="calculator w-[50%] md:w-[30%] m-auto h-full">
+    <input
+      className="calculator-screen text-right text-4xl font-bold bg-gray-400 w-full mt-4 outline-none pr-1 text-white border-b border-b-white"
+      type="text"
+    />
+    <div className="calculator-keys grid grid-cols-4 gap-[0.1rem]">
+      <button type="submit" className="key" name="AC">
+        AC
```
Comment (codepantha) [inline src/components/Calculator.js:9]: - [ ] According to project requirements, the `<Calculator />` component should have at least 1 child component. Please implement this requirement 👍 
**HINT:** You can create a `<ResultDisplay />` or a reusable `<Button />` component or both (preferably) 😎

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @wwwdrt 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ Created `<Calculator />` component 📌 
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._
- [ ] Please remove all the boilerplate code that got created with CRA
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by skyv26 — APPROVED** [review]
## Hi @wwwdrt, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well-structured project ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**